pkg/types/stack.go

```diff
@@ -106,3 +107,14 @@ func (s *Stack) IsOldFileStructure() (bool, error) {
 		return false, nil
 	}
 }
+
+func (s *Stack) ConcatenateEnvironmentVars(input map[string]interface{}) map[string]interface{} {
```

Comment:
nit: rename to `ConcatenateWithProvidedEnvironmentVars`

Reply:
renamed